hardhat.config.ts

```diff
@@ -12,11 +12,11 @@ module.exports = {
   },
   networks: {
     starknetLocalhost: {
-      url: "http://localhost:5000"
+      url: "http://host.docker.internal:5000/"
```

Comment:
You will want to change this once the "smarter" plugin version is released which tackles this localhost issue automatically.